content/zh/docs/concepts/storage/storage-classes.md

```diff
@@ -225,14 +225,14 @@ mount options specified in the `mountOptions` field of the class.
 
 If the volume plugin does not support mount options but mount options are
 specified, provisioning will fail. Mount options are not validated on either
-the class or PV, so mount of the PV will simply fail if one is invalid.
+the class or PV, If a mount option is invalid, the PV mount fails.
  -->
 ### 挂载选项
 
 由 StorageClass 动态创建的 PersistentVolume 将使用类中 `mountOptions` 字段指定的挂载选项。
 
-如果卷插件不支持挂载选项，却指定了该选项，则制备操作会失败。
-挂载选项在 StorageClass 和 PV 上都不会做验证，所以如果挂载选项无效，那么这个 PV 就会失败。
+如果卷插件不支持挂载选项，却指定了选项，则制备操作会失败。
+挂载选项在 StorageClass 和 PV 上都不会做验证，所以如果有一个挂载选项无效，那么这个 PV 挂载就会失败。
```

Comment:
```suggestion
挂载选项在 StorageClass 和 PV 上都不会做验证，如果有任何一个挂载选项无效，那么这个 PV 挂载就会失败。
```

Reply:
Your suggestion is better, I will update it.